app.py

```diff
@@ -69,6 +69,21 @@ def inject_template_variables():
     }
 
 
+# Template helpers
+@app.context_processor
+def utility_processor():
```

Comment:
I am not a big fan of passing functions like this in the template. There is no other way of processing the data before sending to the template?

Reply:
I figure these are 'mission critical' for anything other than input values/ display so was better suited to be in the view layer - could definitely add some helper functions somewhere and do it in the template.

Reply:
Actually, I'd have to do it for every metric added so would have to add a `is_{metric}_metric_enabled` for every metric we added to the context... Seems overkill for checking whether an array contains an item.